grafana/README.md

```diff
@@ -0,0 +1,42 @@
+# Prometheus & Grafana for Nessie
+
+## Dev installation for local experiments
+
+These steps are mainly for setting up Prometheus/Grafana/Nessie locally.
+
+* Install Minikube as described in https://minikube.sigs.k8s.io/docs/start/
+* Install Helm as described in https://helm.sh/docs/intro/install/
+* Start Minikube cluster: `minikube start`
+* Create K8s Namespace: `kubectl create namespace nessie-dev`
+
+## Prometheus & Grafana & Nessie installation
+
+We use the `kube-prometheus-stack` which includes Prometheus & Grafana.
+To scrape Nessie metrics we use a [ServiceMonitor](https://github.com/projectnessie/nessie/blob/main/helm/nessie/templates/servicemonitor.yaml).
+
+1. Add Prometheus repo:
+```sh
+helm repo add prometheus-community https://prometheus-community.github.io/helm-charts
+helm repo update
+```
+2. Install Prometheus
+```sh
+helm -n nessie-dev install prometheus prometheus-community/kube-prometheus-stack
+```
+3. Install Nessie (see also https://github.com/projectnessie/nessie/blob/main/helm/README.md)
```

Comment:
nit: i noticed that the linked readme uses `nessie-ns` as namespace and `nessie-dev` in other places, maybe should be unified

Reply:
updated